docs/user-guide.md

```diff
@@ -50,10 +51,11 @@ As described in Configuration Overview, there are 2 things that you need to cons
 
 For `Gatling docker image`, you can use default image `ghcr.io/st-tech/gatling:latest`, or you can create custom image to use.
 
-For `Gatling load testing related files`, you have 2 options:
+For `Gatling load testing related files`, you have 3 options:
 
 - Create custom image to bundle Gatling load testing files with Java runtime and Gatling standalone bundle package
 - Add Gatling load testing files as multi-line definitions in `.spec.testScenatioSpec` part of `Gatling CR`
+- Set up persistent volumes in `.persistentVolume` and `.persistentVolumeClaim` in `Gatling CR` and load test files from the persistent volumes in Gatling load test files.
```

Comment:
In my understanding, the number of available PersistentVolume is only one. Correct?

Reply:
Correct.  Remove the plural expression for `persistent volumes`.